packages/dev/core/src/Shaders/pbr.fragment.fx

```diff
@@ -672,11 +672,7 @@ void main(void) {
     #endif
 
     #ifdef PREPASS_REFLECTIVITY
-        #if defined(REFLECTIVITY)
-            gl_FragData[PREPASS_REFLECTIVITY_INDEX] = vec4(baseReflectivity.rgb, baseReflectivity.a * writeGeometryInfo);
-        #else
-            gl_FragData[PREPASS_REFLECTIVITY_INDEX] = vec4(0.0, 0.0, 0.0, writeGeometryInfo);
-        #endif
+        gl_FragData[PREPASS_REFLECTIVITY_INDEX] = vec4(toGammaSpace(specularEnvironmentR0), microSurface) * writeGeometryInfo;

```

Comment:
Hello,
To me it doesn't break existing code since reflectivity was already returned in gamma space.
I also find strange that prePass returned gamma values, but if we change to linear this will affect SSR rendering..

Reply:
Agree mostly for back compat despite prefering a linear way as well.

Reply:
What do you think about:
- returning linear reflectivity in both prePass and geomBuffer
- applying toGammaSpace in the SSR postProcess for back compatibility ?